www/lendingbot.js

```diff
@@ -7,10 +7,17 @@ var Day = new Timespan("Day",1);
 var Week = new Timespan("Week",7);
 var Month = new Timespan("Month",30);
 var timespans = [Month, Week, Day, Hour];
-var Coin_Val = 1.00000000;
-var Coin = "BTC";
+var summeryCoinRate = 1.00000000;
+var summeryCoin = "BTC";
```

Comment:
true, it's -5 degrees here... I need summer! :)

I also need to fix the USDT calculation which I broke.